test/functional/assertion-helper.js

```diff
@@ -66,38 +71,49 @@ exports.errorInEachBrowserNotContains = function errorInEachBrowserNotContains (
     }
 };
 
-exports.checkScreenshotsCreated = function checkScreenshotsCreated (withoutScreenshot, count, customPath) {
-    var screenshotsPath     = '___test-screenshots___';
-    var expectedCount       = count || 2;
-    var screenshotDirExists = isDirExists(screenshotsPath);
+exports.checkScreenshotsAreNotCreated = function () {
```

Comment:
I don't expect any actions (like dir removing) from the this function by its name. Moreover we have the `checkScreenshotsCreated` function. I expect just
```js
checkScreenshotsAreNotCreated  () {
    return !checkScreenshotsCreated();
}
```

So, I suppose we should rename this. Maybe just `clearScreenshotsDir`?